hr_timesheet_filter_adj/__manifest__.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017-2020 Quartile Limited
```

Comment:
It is a new module, we can just simple make it `# Copyright 2020 Quartile Limited`